docs/services/slam/cartographer/_index.md

```diff
@@ -0,0 +1,366 @@
+| `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | |
+| `num_range_data` | Number of measurements in each submap. | Optional | `100` | |
+| `missing_data_ray_length` | Replaces the length of ranges that are further than max_range with this value. | Optional | `25` | Nominally set to max length. |
+| `max_range` | Maximum range of valid measurements. | Optional | `25` | |
+| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | |
+| `max_submaps_to_keep` | Number of submaps to use and track for localization. | Optional | `3` | Only for [LOCALIZING mode](#mapping-modes). |
+| `fresh_submaps_count` | Length of submap history considered when running SLAM in updating mode. | Optional | `3` | Only for [UPDATING mode](#mapping-modes). |
+| `min_covered_area` | The minimum overlapping area, in square meters, for an old submap to be considered for deletion. | Optional | `1.0` | Only for [UPDATING mode](#mapping-modes). |
+| `min_added_submaps_count` | The minimum number of added submaps before deletion of the old submap is considered. | Optional | `1` | Only for [UPDATING mode](#mapping-modes). |
+| `occupied_space_weight` | Emphasis to put on scanned data points between measurements. | Optional | `20.0` | Normalized with translational and rotational. |
+| `translation_weight` | Emphasis to put on expected translational change from pose extrapolator data between measurements. | Optional | `10.0` | Normalized with occupied and rotational. |
+| `rotation_weight` | Emphasis to put on expected rotational change from pose extrapolator data between measurements. | Optional | `1.0` | Normalized with translational and occupied. |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## View the Map
+
+After saving your config and connecting to your robot, navigate to the **control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+
+Change the **Refresh frequency** to your desired frequency.
+If in Live Data mode, move your Rplidar around slowly.
+Watch a map start to appear.
+
+{{% alert title="Note" color="note" %}}
+It might take a couple of minutes before a map is displayed.
+Keep moving the camera slowly within your space and wait for the map to visualize.
+{{% /alert %}}
+
+## Troubleshooting
+
+### Tip: Lidar Mounting for the Viam Rover
+
+If you have a Viam Rover and need a mount for your RPLidar, you can 3D print an adapter plate.
+The STL file for an adapter plate is available on [GitHub](https://github.com/viamrobotics/VR1-22-A001/blob/master/CAD/RPIidar_adapter.STL).
+
+### Known Issues
+
+#### Maps JPEG not appearing in UI
+
+When generating a larger map, it can take a while for the Cartographer service to return the `JPEG` map.
+
+Reducing the frequency the Cartographer service returns the map by adjusting **Refresh frequency** should help the JPEG visualization to appear consistently.
```

Comment:
[nit] The maps are now not JPEG anymore. In the UI, they're titled "View SLAM Map (2D)". The same issue is true for 3D, though. I will suggest an alternative text - feel free to modify it, though!

```suggestion
#### Maps not appearing in UI

When generating a larger map, the Cartographer service can take a while to return and visualize the map.

Reducing the frequency the Cartographer service returns the map by adjusting **Refresh frequency** should help the map visualization to appear consistently.
```

Reply:
lgtm done!